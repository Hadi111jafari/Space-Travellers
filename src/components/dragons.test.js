import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import Dragons from './dragons.js';

describe('Dragons component', () => {
  test('component match with the snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <Dragons />
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
