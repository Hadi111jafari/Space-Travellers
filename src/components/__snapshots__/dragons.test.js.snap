// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dragons component component match with the snapshot 1`] = `
<section>
  <ul
    className="dragonsList"
  />
</section>
`;
